ninjalooter/constants.py

```diff
@@ -1,15 +1,20 @@
 # Player Classes
-BARD = "Bard"
-CLERIC = "Cleric"
-DRUID = "Druid"
-ENCHANTER = "Enchanter"
-MAGICIAN = "Magician"
-MONK = "Monk"
-NECROMANCER = "Necromancer"
-PALADIN = "Paladin"
-RANGER = "Ranger"
-ROGUE = "Rogue"
-SHADOW_KNIGHT = "Shadow Knight"
-SHAMAN = "Shaman"
-WARRIOR = "Warrior"
-WIZARD = "Wizard"
+BARD            = "Bard"
```

Comment:
You've added a bunch of whitespace here that needs to be removed -- this will cause pep8 checks to fail.